### ISSUE TITLE
add more diverse color themes

### ISSUE THREAD

**Issue by ghost** [Nik-Nothing/binfetch#5]

**Comment by ghost** [comment]
note: maybe colors dependent on checksums, to make art differ even more

**Comment by xplshn** [comment]
Yes, any way to change the color scheme? Or use black/white, or maybe disable the Ascii art entirely?
For know I've edited the src code and just set other colors.

**Comment by Im-0xea** [comment]
jup, look at https://github.com/Im-0xea/binfetch/tree/main/cfg

you can overwrite the default with -c / --config or by replacing binfetch.cfg in .config/binfetch or if not present $PREFIX/share/binfetch
